src/Generator/Util.fs

```diff
@@ -0,0 +1,72 @@
+module Util
+
+open Types
+
+let inline (</>) path1 path2 = System.IO.Path.Combine(path1, path2)
+
+type MaybeBuilder() =
```

Comment:
Is it this going to be to do CE? such as maybe { ... } ?

Reply:
Yes it is, see a few lines below.

Reply:
Except for when you define the maybe builder, I haven't seen a use for it which triggered my question. Maybe there's something I'm missing??

Reply:
I checked this, the maybe builder is still in use.